gradle/dependencies.gradle

```diff
@@ -51,7 +51,7 @@ ext.deps = [
             runner: 'com.android.support.test:runner:0.5',
         ],
     ],
-    kotlinPoet: 'com.squareup:kotlinpoet:1.0.1',
+    kotlinPoet: 'com.squareup:kotlinpoet:1.1.0-SNAPSHOT',
```

Comment:
Releasing soon

Reply:
Any rough ETA? Soon like we can merge and update tomorrow, or should we just keep this open until release? Snapshot blocks release and we probably need another one soon.

Reply:
My plan is tomorrow morning

Reply:
looks like its released now?

Reply:
Yes